src/CMakeLists.txt

```diff
@@ -172,45 +176,18 @@ if(STATIC_LINKING)
   target_link_libraries(runtime ${LIBBPF_LIBRARIES})
   target_link_libraries(runtime ${LIBBCC_LOADER_LIBRARY_STATIC})
 
+  find_package(LibLzma)
```

Comment:
Is this always necessary? Or just for the case when LLDB is found?

Reply:
It's always necessary for static build, liblzma is required by libbcc.
See https://github.com/bpftrace/bpftrace/actions/runs/8330340247/job/22794747512?pr=3042#step:3:623